src/ui/public/partials/saved_object_finder.html

```diff
@@ -1,42 +1,70 @@
-<form role="form">
-  <div class="form-group finder-form">
-    <div class="finder-form-options">
-      <a class="small" ng-click="finder.manageObjects(finder.properties.name)">manage {{finder.properties.nouns}}</a>
+<form role="form" class="form-inline">
+  <div class="container-fluid">
+    <div class="saved-object-finder-row row">
+      <div class="saved-object-finder-input input-group form-group finder-form col-md-8">
+        <span class="search-icon input-group-addon">
+          <i class="fa fa-search"></i>
+        </span>
+        <input
+          input-focus
+          ng-model="filter"
+          ng-attr-placeholder="{{finder.properties.nouns | capitalize}} Filter..."
+          ng-keydown="finder.filterKeyDown($event)"
+          class="form-control"
+          name="filter"
+          type="text" />
+      </div>
+      <div class="saved-object-finder-column col-md-2">
+        <span>{{finder.hitCount}} of {{finder.hitCount}}</span>
+      </div>
+      <div class="saved-object-finder-column-manage col-md-2">
+        <a class="small" ng-click="finder.manageObjects(finder.properties.name)">Manage {{finder.properties.nouns | capitalize}}</a>
+      </div>
     </div>
-    <div class="clearfix visible-xs-block"></div>
-    <input
-      input-focus
-      ng-model="filter"
-      ng-attr-placeholder="{{finder.properties.noun}} Filter"
-      ng-keydown="finder.filterKeyDown($event)"
-      class="form-control"
-      name="filter"
-      type="text"
-      autocomplete="off" />
-
-    <span class="finder-hit-count"><strong>{{finder.hitCount}}</strong> {{finder.hitCountNoun()}}</span>
   </div>
 </form>
-<paginate list="finder.hits" per-page="{{perPage}}">
-  <ul
-    class="list-group list-group-menu"
+<paginate list="finder.hits" per-page="20" class="paginate-style">
+  <ul class="li-striped list-group list-group-menu"
     ng-class="{'select-mode': finder.selector.enabled}">
 
+    <li class="list-header list-group-item list-group-menu-item">
+      <ul class="li-not-striped">
```

Comment:
this feels off. Can we create a style for the paginate directives and apply it, instead of making classes that aren't reused outside of paginate?